src/rewriter.coffee

```diff
@@ -263,6 +263,20 @@ exports.Rewriter = class Rewriter
             stack.pop()
         start = stack.pop()
 
+      inControlFlow = =>
+        seenFor = @findTagsBackwards(i, ['FOR']) and @findTagsBackwards(i, ['FORIN', 'FOROF', 'FORFROM'])
+        controlFlow = seenFor or @findTagsBackwards i, ['WHILE', 'UNTIL', 'LOOP', 'LEADING_WHEN']
+        return no unless controlFlow
+        isFunc = no
+        tagCurrentLine = token[2].first_line
+        @detectEnd i,
```

Comment:
Speaking of line continuations, shouldn’t there be a `\` here? I don’t like relying on implicit line continuation.

Reply:
You mean for these cases
```coffeescript
for a in b \
-> c

for a in b \
do -> c

for \
a \
in \
b \
do \
-> \
```

Reply:
we've accepted `,` since basically 2010. It's set to stay now :).

Reply:
Oh, that one.

Reply:
Fine. 😞